kubernetes/helm/resources/gateway-api-operator-demo/README.md

```
@@ -0,0 +1,190 @@
+
+   ```bash
+   kubectl get httproute hello-api -n gateway-api-demo -o yaml
+   ```
+
+7. **Invoke the routed API with `curl`** (through gateway-runtime Service):
+
+   ```bash
+   curl --request GET \
+  --url https://localhost:8443/hello-context/hello \
+  --header 'Accept: application/json' -k
+   ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)